### PR TITLE
ncutil 2.4 (new formula)

### PR DESCRIPTION
**Pull request by jacobsalmela**
Add NCutil to Homebrew.  A utility to modify OS X's Notification Center from the command line.

### CODE REVIEW

File: Library/Formula/ncutil.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Ncutil < Formula
+  desc "Modify OS X's Notification Center: Add/remove apps, get/set alert styles, suppress notifications, etc."
+  homepage "https://github.com/jacobsalmela/NCutil"
+  url "https://github.com/jacobsalmela/NCutil/archive/2.4.tar.gz"
+  version "2.4"
+  sha256 "ebea1fe1163e401dd4e7833416001dbf9d60614237700755b55b46d0abf423bc"
+
+  def install
+	bin.install "NCutil.py" => "ncutil"
```
Comment (mikemcquaid) [inline Library/Formula/ncutil.rb]: Please use two-character indent.

File: Library/Formula/ncutil.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  test do
+	system "#{bin}/ncutil", "--help"
```
Comment (mikemcquaid) [inline Library/Formula/ncutil.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by mikemcquaid** [comment]
Various style/audit failures: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/37918/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_ncutil___strict___online/

File: Library/Formula/ncutil.rb (hunk 2 above)
Comment (jacobsalmela) [inline Library/Formula/ncutil.rb]: Wilco.

**Comment by mikemcquaid** [comment]
Test is failing on 10.9: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/37948/version=mavericks/testReport/junit/brew-test-bot/mavericks/test_ncutil/

**Comment by jacobsalmela** [comment]
@mikemcquaid, yeah, I have been trying to figure out why it is producing that error.  Sorry for the delay.

**Comment by jacobsalmela** [comment]
It works if run locally.  I can try a different test.

**Comment by mikemcquaid** [comment]
Looks like it still fails on Mavericks (10.9). Does this definitely work there?

**Comment by jacobsalmela** [comment]
I did a clean install of Mavericks and ran NCutil just fine.  I'm not sure what is making it fail.  I have been trying to troubleshoot for a while, but it seems to run fine on Mavericks when installed locally.

**Comment by bfontaine** [comment]
Does `brew test --sandbox ncutil` work for you?

**Comment by jacobsalmela** [comment]
Yes, that command executes fine.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by jacobsalmela** [comment]
It talks about a NoneType error on Mavericks (so says BrewTestBot) but it works well when run manually.

**Comment by mikemcquaid** [comment]
@jacobsalmela Run manually on Mavericks? We'll need to sort out the test failure before this can be included.

**Comment by MikeMcQuaid** [comment]
Passing on this as-is, please reopen on https://github.com/homebrew/homebrew-core if desired.

**Pull request closed without merging**